Sources/IndexStoreDB/IndexStoreDB.swift

```diff
@@ -109,6 +109,17 @@ public final class IndexStoreDB {
 
 }
 
+public final class SymbolRelation {
+  let value: indexstoredb_symbol_relation_t
+
+  public lazy var roles = SymbolRole(rawValue: indexstoredb_symbol_relation_get_roles(value))
```

Comment:
Please add type annotations (`: SymbolRole`) to any class/struct properties.  It makes the API more explicit and can improve compile times.